.travis.yml

```diff
@@ -32,8 +32,8 @@ env:
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
   - VERSION="6.1" INCLUDE="test_module,second_module"
   - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="18" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - VERSION=master LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="18" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
-  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="18" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+  - VERSION=master LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="25" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
```

Comment:
I need to test `master` word in pylint (Different to number 7.0, 6.1...)
The trouble in the past was with TESTS="1" but I have TESTS="0"